applications/post-editing.md

```diff
@@ -0,0 +1,21 @@
+---
+description: Human editing of machine translation output
+---
+
+# Post-editing
+
+Human **Post-editing** is the review and correction of machine translation output. When post-editing, the human translator checks the original text and fixes any error made by the machine translation.
+
+Post-editing is a replacement of human translation as it has faster turnaround times, is cheaper, and can process larger amounts of information than the traditional human translation process.
+
+Over the 2010s, with rising machine translation quality, it became the dominant human translation workflow.
+
+### Flavours
+With **light post-editing**, the goal is that readers understand the idea of a text. Light post-editing provides the gist of a message. It is usually aimed at internal communications.
+
+With **full post-editing**, the goal is to match human quality. Clients may request full post-editing services for publishable information.
+
+### Metrics
+**Post-editing distance** provides a percentage of how much the text has changed from the initial to the edited version. Percentages closer to 0% will mean that the machine translation output required minimum editing. Light and full post-editing are related to expected changes percentage.
```

Comment:
Just say what it is

**Post-editing distance** is a measure of how much the human translator post-edited the machine translation.  It's both a metric for a specific translation, and also used in aggregate.

It can be length-normalised and given as a percentage.